docs/setup_vscode_macos.md

```diff
@@ -475,7 +475,7 @@ First, you should already have the `C/C++` extension installed ([Instructions](#
 
 Next, open VS Code's Command Palette with `View > Command Palette` or with the keyboard shortcut `cmd + shift + P`.  Search for and select `C/C++: Edit Configurations (JSON)`.  This will open the file `c_cpp_properties.json`.
 
-<img src="images/vscode150.png" width="480px" />
+<img src="images/vscode_macos_150.png" width="480px" />
```

Comment:
I think line 505 below (the one that mentions WSL) is not relevant to macOS. Can you remove it?